integration_tests/pages/authManageDetails.ts

```diff
@@ -2,6 +2,6 @@ import Page from './page'
 
 export default class AuthManageDetailsPage extends Page {
   constructor() {
-    super('account-details', false)
+    super('account-details', { axeTest: false })
```

Comment:
The `Page` constructor now has an options object as its 2nd arg - see `Page` below